pkg/monitors/vsphere/service/gateway.go

```diff
@@ -14,16 +14,20 @@ import (
 
 // A thin wrapper around the vmomi SDK so that callers don't have to use it directly.
 type IGateway interface {
+	IPerfQuerier
```

Comment:
In the future, don't use the Java `I` prefix for interfaces.  Just call it a `PerfQuerier`.

Reply:
Done.